### PR TITLE
Implement entry attribute referrals API V2

### PR DESCRIPTION
**Pull request by syucream**
Replace `get_attr_referrals`  with a new endpoint in API V2.

### CODE REVIEW

File: entry/api_v2/views.py [hunk 1]
```diff
@@ -359,3 +362,40 @@ def post(self, request, entity_id: int):
...
+        else:
+            entity_attr = EntityAttr.objects.filter(id=attr_id).first()
+        if not entity_attr:
+            raise NotFound(f"not found matched attribute or entity attr: {attr_id}")
+
+        conditions = {"is_active": True}
+        if keyword:
+            conditions["name__icontains"] = keyword
+
+        # TODO support natural sort?
```
Comment (syucream) [inline entry/api_v2/views.py:391]: In API V1, it performs `natsort`. its not replacable in queryset ... is it required? should we perform it on the client side?

**Review by hinashi — APPROVED** [review]
LGTM!

File: entry/api_v2/views.py (hunk 1 above)
Comment (userlocalhost) [reply entry/api_v2/views.py:391]: It's OK to implement with your work. Let's fix it when any problem will be emerged 👍

**Review by userlocalhost — APPROVED** [review]
LGTM 👍 Thank you for your hard work!!

**Pull request merged**